### ISSUE TITLE
[FUSION] cryptomatte working partially  

### ISSUE THREAD

**Issue by ballobello** [Psyop/Cryptomatte#101]
Hi! I´m new here and trying to test Cryptomatte in Fusion 9.02

After I tried Reactor cryptomatte, where I can add the node, but tools aren´t visible. I swap to do it manually, I read carefully the instructions and I now I can say cryptomatte is working, I can add the node, but when I using it with my multi-exr from Arnold/Maya, I got this message when I connect the node. I´m a bit frustrated right now, what I did wrong?
With the Bunny sample images works like a charm

```
> ...agic Design\Fusion\Modules\Lua\cryptomatte_utilities.lua:809: attempt to index a nil value
> stack traceback:
> 	...agic Design\Fusion\Modules\Lua\cryptomatte_utilities.lua:809: in function 'get_manifest_string'
> 	...agic Design\Fusion\Modules\Lua\cryptomatte_utilities.lua:730: in function 'initialize'
> 	...agic Design\Fusion\Modules\Lua\cryptomatte_utilities.lua:912: in function 'create_cryptomatte_info'
> 	...ng\Blackmagic Design\Fusion\Fuses\Matte\cryptomatte.fuse:583: in function <...ng\Blackmagic Design\Fusion\Fuses\Matte\cryptomatte.fuse:568>
> Cryptomatte1 failed at time 1
```

**Comment by acrux7245** [comment]
I am having same exact error with Vray and 3ds max. Weird thing is it was working fine up until about a week ago. Error on multiple machines on free and payed Fusion 9.

**Comment by podaliac** [comment]
hello.  i'm having the same problem as the original post.
i installed cryptomatte using reactor.  i've also tried installing it manually.
anyone know what to do?
thanks

**Comment by omento** [comment]
Everything seems to be working fine over here on Fu16/MtoA 3.1.2.1. Installed through Reactor.

**Comment by cedricduriau** [comment]
Fixed in cedricduriau#43
